.github/workflows/add-rebase-label.yml

```diff
@@ -1,14 +1,14 @@
-name: Add rebase label

```

Comment:
@srividya0208 Could you properly rebase without these spurious changes unrelated to the change in question?